Fix/make council votes default expanded

jonlin14 opened the pull request:
### Link to Relevant Issue
https://github.com/CouncilDataProject/cdp-frontend/issues/163

### Description of Changes
Reworks MeetingVoteTableRow to have council votes always be expanded if 5 or less council members are present.
If more than 5 council members are in the meeting, only show the first 5, then render a clickable button to show more. 

### Link to Forked Storybook Site
https://jonlin14.github.io/cdp-frontend/#/events/756436a9130b -- Five council members, no button as expected
https://jonlin14.github.io/cdp-frontend/#/events/de58242d129b -- Seven council members, shows an expand button and hides any council members past 5 as expected.

evamaxfield commented:
Hey @jonlin14! Thanks for doing this!!

This looks really good. I only have two requests from the "design side":
1. Can we add a bit of spacing between the button and the last row of text? I manually edited the HTML in my browser and I think a `margin-top: 10px` looks nice.

2. Can you remove the text from the page that says: "Click each row to see how council members voted" because that is now not needed!
![image](https://user-images.githubusercontent.com/17132317/216134234-8abfe4b3-8fbc-4490-9aac-4ab8184076dd.png)

I will let @BrianL3 review the actual code but thanks a lot!

jonlin14 commented:
Great, removed the click to see how each member voted text and added the margin top. Also made a fix to replicate the current mobile behavior. The original links in the pull request description reflects these changes.

BrianL3 commented:
Nice! Code looks good.
